WalletWasabi/WabiSabi/Client/Banning/CoinPrison.cs

```diff
@@ -78,6 +82,24 @@ private void ToFile()
 		File.WriteAllText(FilePath, json);
 	}
 
+	private static DateTimeOffset ReduceBanningTimeIfNeeded(DateTimeOffset bannedUntil)
```

Comment:
Please add some comments with `///` about why this is required

Reply:
Added.